core/src/main/java/eu/neverblink/jelly/core/utils/LogicalStreamTypeUtils.java

```diff
@@ -128,4 +131,43 @@ public static <TNode, TDatatype, TTriple> List<TTriple> getRdfStaxAnnotation(
             )
         );
     }
+
+    /**
+     * Returns an RDF-STaX annotation for the logical stream type, in RDF. The annotation simply states that
+     * &lt;subjectNode&gt; has a stream type usage, and that stream type usage has this stream type.
+     * <p>
+     * Example in Turtle for a flat triple stream:
+     * &lt;subjectNode&gt; stax:hasStreamTypeUsage [
+     *     a stax:RdfStreamTypeUsage ;
+     *     stax:hasStreamType stax:flatTripleStream
+     * ] .
+     * <p>
+     * This is a wrapper for the {@link #getRdfStaxAnnotation(ProtoDecoderConverter, TripleMaker, LogicalStreamType, Object)},
+     * that assumes that the decoder implements the {@link ProtoDecoderConverter} interface and {@link TripleMaker} interface.
+     *
+     * @param <TNode> the type of RDF nodes
+     * @param <TDatatype> the type of RDF triples
+     * @param <TTriple> the type of RDF triples
+     * @param <TEncoder> the type of RDF encoders
+     * @param <TDecoder> the type of RDF decoders
+     * @param factory the converter factory to use for creating RDF nodes
+     * @param logicalType the logical stream type
+     * @param subjectNode the subject node to annotate
+     * @throws IllegalArgumentException if the logical stream type is not supported
+     * @return the RDF-STaX annotation
+     */
+    public static <
+        TNode,
+        TDatatype,
+        TTriple,
+        TEncoder extends ProtoEncoderConverter<TNode>,
+        TDecoder extends ProtoDecoderConverter<TNode, TDatatype> & TripleMaker<TNode, TTriple>
+    > List<TTriple> getRdfStaxAnnotation(
```

Comment:
Please add a tiny, tiny test to make sure it works.

Reply:
what is there to break?

Reply:
Lack of the method :) no test -> someone could remove it accidentally. Or break in some funny way. Just make a tiny dumb test.